src/tsickle.ts

```diff
@@ -227,8 +227,33 @@ function addClutzAliases(
     // default is a keyword in typescript, so the name of the export being default means that it's a
     // default export.
     if (e.name === 'default') return false;
+
     // Otherwise check that some declaration is from the local module.
-    return e.declarations.some(d => d.getSourceFile() === origSourceFile);
+    for (const d of e.declarations) {
+      // This check helps us filter out export * rewritten exports. It appears,
+      // those point to the source file without any need to get through the
+      // getExportSpecifierLocalTargetSymbol API below.
+      if (d.getSourceFile() !== origSourceFile) {
+        return false;
+      }
+
+      // export specifiers are ambigious - import {X}; export {X} vs
+      // export {X}, but everything else is a local export.
```

Comment:
I couldn't quite parse this,   export {X} vs export {X}?  Add some newlines?

Reply:
reworded the comments.